benches/primitives.rs

```diff
@@ -58,7 +58,7 @@ fn sector_150(c: &mut Criterion) {
         Sector::with_center(
             BOUNDING_BOX.center(),
             BOUNDING_BOX.size.width,
-            -30.0.deg(),
+            -(30.0.deg()),
```

Comment:
Is this a clippy fix or why did you include this change in this PR?

Reply:
> Is this a clippy fix or why did you include this change in this PR?

I ran `cargo clippy --fix` in the repo.

Reply:
OK, I will add them to `CHANGELOG.md`.